sdk/src/cose_validator.rs

```diff
@@ -814,6 +828,76 @@ fn get_timestamp_info(sign1: &coset::CoseSign1, data: &[u8]) -> Result<TstInfo>
     Err(Error::NotFound)
 }
 
+#[async_generic(async_signature( th: &dyn TrustHandlerConfig, chain_der: &[Vec<u8>], cert_der: &[u8], validation_log: &mut impl StatusTracker))]
+#[allow(unused)]
+fn check_trust(
+    th: &dyn TrustHandlerConfig,
+    chain_der: &[Vec<u8>],
+    cert_der: &[u8],
+    validation_log: &mut impl StatusTracker,
+) -> Result<bool> {
+    // is the certificate trusted
+
+    let verify_result: Result<bool> = if _sync {
+        #[cfg(not(feature = "openssl_sign"))]
```

Comment:
change these to "openssl" so we will still have them when openssl_sign isn't required.